docs/tutorials/notebooks/Monge_Gap.ipynb

```diff
@@ -0,0 +1,734 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    from typing_extensions import Literal
</pre><p>This can be imported from <code>typing</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>